### ISSUE TITLE
Execution error while execute any .mesh file

### ISSUE THREAD

**Issue by biswajitcsecu** [mfem/mfem#905]
MFEM abort: Unknown input mesh format: MeshVersionFormatted 2
 ... in function: void mfem::Mesh::Loader(std::istream&, int, std::__cxx11::string)
 ... at line 2715 of file: /home/nebex/Desktop/mfem/src/mesh/mesh.cpp

I am not sure how I resolve???????????

**Comment by mlstowell** [comment]
Hello, @biswajitcsecu ,
Are you getting this error message from one of our example programs or something else?  Do you get this message with our mesh files from the "data" directory?
Best wishes,
Mark

**Comment by biswajitcsecu** [comment]
I put the test data i.e. a mesh file in my present project folder but I getting same errors???

**Comment by mlstowell** [comment]
Hi, @biswajitcsecu ,
We are going to need more information before we can help you.  Please start by answering the two questions I asked yesterday.
Best wishes,
Mark

**Comment by jonesholger** [comment]
Looks like a Medit file Ref:  https://github.com/ISCDtoolbox/Medit, and https://github.com/nschloe/meshio/issues/291

**Comment by biswajitcsecu** [comment]
Hi, jonesholger,

Thank you very much for your kind response.

**Comment by jonesholger** [comment]
@biswajitcsecu  No problem! We'll go ahead and close this issue, as we like to dispatch them in a reasonable time frame to keep the list manageable. Feel free to re-open, or to open another issue, as you progress and have more questions.
